src/main/java/seedu/address/model/Name.java

```diff
@@ -39,7 +39,7 @@ public Name(String name) {
     public static class EmptyName extends Name {
         public static final Name EMPTY_NAME = new EmptyName();
         public EmptyName() {
```

Comment:
think the constructor inside can be private 
```suggestion
        private EmptyName() {
```